packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -36,6 +37,7 @@ export function Disclosure({
   const [isMounted, setMount] = useState(false);
   const [titleRef, { exactWidth }] = useResizeObserver<HTMLDivElement>();
   const isBelowBreakpoint = exactWidth && exactWidth < Breakpoints.small;
+  const isTitleString = typeof title === "string";
```

Comment:
we're doing this same check twice. once up here and then again inside `DisclosureTitle`.

not the end of the world, but since `DisclosureTitle` is an internal component, we might as well add a prop that can leverage the check we already did

maybe something like `isTitleString: boolean`

if this was a component/prop being exposed to consumers I'd be tempted to make it more behavior focused rather than implementation details but since it's not, I don't think we need to fuss about that.